pkgs/desktops/xfce4-13/default.nix

```diff
@@ -82,7 +82,7 @@ makeScope newScope (self: with self; {
 
   xfce4-volumed-pulse = callPackage ./xfce4-volumed-pulse { };
 
-  xfce4-whiskermenu-plugin = callPackage ./xfce4-whiskermenu-plugin { };
```

Comment:
This was just added in #44317. Why  remove it now?

Reply:
it should not be removed :(
Actual PR is https://github.com/NixOS/nixpkgs/pull/43778, I am just trying to split to to small pieces to get some traction.
But it results to merge collisions :(